plutus-tx/src/PlutusTx/Prelude.hs

```diff
@@ -29,11 +38,15 @@ module PlutusTx.Prelude (
     check,
     -- * Booleans
     module Bool,
-    -- * Int operators
+    -- * Integer numbers
+    Integer,
+    toInteger,
+    fromIntegral,
```

Comment:
Probably shouldn't export `toInteger` or `fromIntegral`, not sure those will work in general.

Reply:
More generally, we don't have any other number types apart from `Integer`!